modules/@apostrophecms/doc-type/index.js

```diff
@@ -186,99 +186,20 @@ module.exports = {
             self.emit('afterDelete', req, previous, { checkForChildren: false });
           }
         },
-        deduplicateArchive(req, doc) {
-          const deduplicateKey = doc.aposDocId;
-          if (doc.parkedId === 'archive') {
-            // The primary archive itself should not deduplicate
-            return;
+        async deduplicateArchive(req, doc) {
+          const $set = await self.deduplicate(req, doc);
```

Comment:
More functional approach: the shared "deduplicate" method has no side effects.

Reply:
Since this isn't doing the duduplication, I suggest `getDeduplicationSet` or `deduplicationSet`